### DISCUSSION TITLE
How do you use compundVariants?

### DISCUSSION THREAD

**Discussion by heyitsarpit** [joe-bell/cva#30] [Q&A] [answered]
I'm not sure how `compundVariants` is used, it seems like a predefined combination of `variants` that can can be passed to the component, however, why are they not named objects and instead an array? How do I access `compundVariants` in props etc...?

**Comment by joe-bell** [comment, marked as answer]
> it seems like a predefined combination of variants

That's it in a nutshell

`compoundVariants` are styles set when a combination of previously specified variants are applied; you use the props in the same way
